lib/taiko.js

```diff
@@ -858,7 +858,10 @@ async function click(selector, options = {}, ...args) {
         options = {};
     }
     if (isSelector(selector) || isString(selector) || Number.isInteger(selector)) {
-        await _click(selector, options, ...args);
+        const clickCount = options.clickCount || 1;
```

Comment:
doing `_click` so many times will be costlier as getting element, proximity search, finding if it covered, scrolling element into view  are all done in it.. guess it will be enough if `waitForMouseActions` is done as many times as click count. Can have a different name to differentiate between `doubleClick` and `clickCount` option